src/Routers/routes/PatientRoutes.tsx

```diff
@@ -70,14 +71,8 @@ const PatientRoutes: AppRoutes = {
       />
     </Suspense>
   ),
-  "/patient/:patientId/drawings/:drawingId": ({ patientId, drawingId }) => (
-    <Suspense fallback={<Loading />}>
-      <ExcalidrawEditor
-        associatingId={patientId}
-        associating_type="patient"
-        drawingId={drawingId}
-      />
-    </Suspense>
+  "organization/:id/patient/:patientId/drawings": ({ patientId }) => (
+    <PatientDrawingTab patientId={patientId} />
```

Comment:
is this right? 🤔 

have you tested these out?

https://github.com/user-attachments/assets/7f587816-7b8f-4046-bb7b-00013c7d10cb